pipelines/controller/src/main/java/org/openmrs/analytics/metrics/TextFormat.java

```diff
@@ -21,27 +21,19 @@
 import java.io.Writer;
 import java.util.Enumeration;
 
-public class TextFormat {
+class TextFormat {
 
   /** Content-type for Data Pipes text version 0.0.1. */
   public static final String CONTENT_TYPE_001 = "text/plain; version=0.0.1; charset=utf-8";
 
-  /** Write out the text version of the given MetricFamilySamples. */
+  /**
+   * Write out the text version of the given MetricFamilySamples. This is inspired from the
+   * prometheus implementation <a
+   * href="https://github.com/prometheus/client_java/blob/main/simpleclient_common/src/main/java/io/prometheus/client/exporter/common/TextFormat.java#L66">TextFormat</a>
```

Comment:
nit: To make this shorter, maybe just mentioned the name of the class (i.e., `TextFormat`) and only include a link to the root, i.e., `https://github.com/prometheus/client_java`.

Reply:
Done